### PR TITLE
Update the logic for comparing values with the initial values

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: #444

### CODE REVIEW

File: src/components/FormPanel/FormPanel.tsx [hunk 1]
```diff
@@ -772,13 +773,12 @@ export const FormPanel: React.FC<Props> = ({
...
   const isUpdated = useMemo(() => {
-    for (const element of elements) {
-      if (element.value !== initial[element.id]) {
-        return true;
-      }
-    }
-
-    return false;
+    return elements
+      .map((element) => {
```
Comment (asimonok) [inline src/components/FormPanel/FormPanel.tsx:777]: please revert to the previous version with `for` and just update `if` statement

**Review by asimonok — CHANGES_REQUESTED** [review]

File: src/components/FormPanel/FormPanel.tsx (hunk 1 above)
Comment (vitPinchuk) [reply src/components/FormPanel/FormPanel.tsx]: done

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**